epregressions/energyplus.py

```diff
@@ -38,13 +38,17 @@ def execute_energyplus(e_args: ExecutionArguments):
 
     # Save the current path so we can go back here
     start_path = os.getcwd()
+
+    std_out = b""
+    std_err = b""
```

Comment:
Keep track of standard output and standard error from every subprocess call so we can watch for unexpected changes in output.